layouts/awesome/single.html

```
@@ -1,61 +1,4 @@
-        function toggle(){L[key]=!L[key];try{localStorage.setItem(k,JSON.stringify(L));}catch(e){}render();btn.classList.remove('pop','spark');void btn.offsetWidth;btn.classList.add('pop','spark');}
-        render();
-        btn.addEventListener('click',function(e){e.preventDefault();e.stopPropagation();toggle();});
-        btn.addEventListener('keydown',function(e){if(e.key==='Enter'||e.key===' '){e.preventDefault();toggle();}});
-      });
-    }
-    if (document.readyState !== 'loading') run(); else document.addEventListener('DOMContentLoaded', run);
-  })();
-</script>
-</body>
-</html>
+{{/*
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)